EU4toV2/Source/EU4World/Relations/EU4Relations.h

```diff
@@ -0,0 +1,24 @@
+#ifndef EU4RELATIONS_H_
+#define EU4RELATIONS_H_
+
+#include "Date.h"
+#include "newParser.h"
+#include "EU4RelationDetails.h"
+#include <map>
+
+namespace EU4
+{
+	class EU4Relations : commonItems::parser
+	{
+	public:
+		EU4Relations(std::istream& theStream);
+		std::map<std::string, EU4RelationDetails> getRelations() const { return relations; }
```

Comment:
This will return a copy of the entire map. Depending on usage, it might be better/more efficient to return ```const std::map<std::string, EU4RelationDetails>&```

Reply:
I have  goal of going over the entire project and fix all raw pointers and calls by value/reference but it wasn't a focus at this time. Fixed this bit though.